apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/R4PatientResourceProvider.java

```diff
@@ -254,13 +255,19 @@ public Patient read(@IdParam IdType patientId, RequestDetails requestDetails) {
   @Trace
   public Bundle searchByLogicalId(
```

Comment:
Also a read endpoint for this class.

Reply:
Regarding open api documentation for read methods. I searched several codebases using open api and hapi fhir, and none of them had the read methods documented. Only search. The problem stems from the idparam class not having a name parameter within its interface. If we could create a class that extends IdParam class, we could add a name attribute, and I think it can work

Reply:
If you launch the server and view swagger you can see the read endpoints. What Im following is how other people using HAPI FHIR and openAPI are doing for reads and searches